7.7.поиск_ошибок_и_ревью_кода/7.7.3.py

```diff
@@ -0,0 +1,12 @@
+count = 0
+p = 1  
+for i in range(1, 11):  
```

Comment:
1. `i` не используется
2. функции `range()` лучше передать один аргумент `10`, чем 2 аргумента `1` и `11`
```suggestion
for _ in range(10):  
```